.github/workflows/code-review.yml

```
@@ -2,27 +1,23 @@
-                     "repository": {
-                       "owner": { "login": "${{ github.repository_owner }}" },
-                       "name": "${{ github.event.repository.name }}"
-                     },
-                     "pull_request": {
-                       "number": ${{ github.event.pull_request.number }},
-                       "diff_url": "${{ github.event.pull_request.diff_url }}"
-                     },
-                     "action": "${{ github.event.action }}"
-                   }'
+      - name: Checkout Repo
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)